### PR TITLE
release a blog - milvus-embraces-nats-messaging

### PR DESCRIPTION
**Pull request by sageanya**

### CODE REVIEW

File: blog/en/milvus-embraces-nats-messaging.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+id: milvus-embraces-nats-messaging.md
+title:
+>
+Optimizing Data Communication: Milvus Embraces NATS Messaging
```
Comment (septemberfd) [inline blog/en/milvus-embraces-nats-messaging.md:5]:
```suggestion

```

File: blog/en/milvus-embraces-nats-messaging.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+---
+id: milvus-embraces-nats-messaging.md
+title:
+>
```
Comment (septemberfd) [inline blog/en/milvus-embraces-nats-messaging.md:4]:
```suggestion

```

File: blog/en/milvus-embraces-nats-messaging.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
+---
+id: milvus-embraces-nats-messaging.md
+title:
```
Comment (septemberfd) [inline blog/en/milvus-embraces-nats-messaging.md:3]:
```suggestion
title: "Optimizing Data Communication: Milvus Embraces NATS Messaging"
```

File: blog/en/milvus-embraces-nats-messaging.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+>
+Optimizing Data Communication: Milvus Embraces NATS Messaging
+author: Zhen Ye
+date: 2023-10-11
```
Comment (septemberfd) [inline blog/en/milvus-embraces-nats-messaging.md:7]:
```suggestion
date: 2023-11-24
```

**Review by septemberfd — APPROVED** [review]

**Comment by septemberfd** [comment]
@sageanya For unknown reasons, the previous way to input the title doesn't work. Let's put the title in the quotation marks. For example: 

title: "Optimizing Data Communication: Milvus Embraces NATS Messaging"

**Comment by sageanya** [comment]
Great, got it. Thanks.


  *   Anya

________________________________

**Pull request merged**